portfolio/src/main/webapp/index.html

```diff
@@ -1,40 +1,44 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Thanks for addressing all of the lint warnings!

Reply:
No problem!